Formula/qt@5.5.rb

```diff
@@ -115,6 +116,16 @@ def install
     inreplace prefix/"mkspecs/qconfig.pri", /\n\n# pkgconfig/, ""
     inreplace prefix/"mkspecs/qconfig.pri", /\nPKG_CONFIG_.*=.*$/, ""
 
+    # In Xcode 8 (especially on Sierra), Qt tries to find xcrun but should be
```

Comment:
The `especially` feels weird here but whichever maintainer pulls this can just remove it then as there's no functional change. Please don't change it @ljones140 to avoid triggering another (lengthy) CI build, thanks.